modules/apps/asset/asset-categories-selector-web/src/main/resources/META-INF/resources/js/SelectCategory.es.js

```diff
@@ -131,7 +136,7 @@ function SelectCategory({
 
 		selectedNodesRef.current = data;
 
-		Liferay.Util.getOpener().Liferay.fire(itemSelectorSaveEvent, {data});
+		openingWindow.Liferay.fire(itemSelectorSaveEvent, {data});
```

Comment:
This is a good example of why we still need custom event support in `openSelectionModal`. OOTB support is only for search container, the most common use case. This is a tree selector, with completely different data representation. /cc @kresimir-coko